helm-charts/doris/values.yaml

```diff
@@ -44,6 +44,8 @@ dorisCluster:
   # password: dDBwLVNlY3JldA==
 
 feSpec:
+  # annotations for fe pods and service
+  # annotations: {}
```

Comment:
This line does not need to be commented. Just remove the ‘#’ symbol while ensuring indentation. Refer to “labels: {}”